components/frontend/src/widgets/Button.js

```diff
@@ -172,7 +172,7 @@ export function AddDropdownButton({ itemSubtypes, itemType, onClick, allItemSubt
                     }
                     value={null} // Without this, a selected item becomes active (shown bold in the menu) and can't be selected again
                 >
-                    <Dropdown.Menu>
+                    <Dropdown.Menu style={{ minWidth: "50em" }}>
```

Comment:
Is this another fix, or needed for this feature?

Reply:
I've enabled line breaks so that long source descriptions don't run off the screen. But then Semantic UI React makes the menu as wide as the trigger which is a button, so that's a bit too small.